### PR TITLE
Refactor change DB collections used for Issue #266

### PR DESCRIPTION
**Pull request by smarkandu**
Covers Issue #266 

Things to note:

- I had to fix the code to take "null" values into account.  For example, if the "status" attribute doesn't exist, it's assumed to be "Unconfirmed", etc.
- I had to change the Client app to store the date of birth in the format specified in:
https://www.w3schools.com/js/js_date_formats.asp
This is in order to be able to create a "Date" object from the given dob string stored in the DB.  After Friday, we should manually change the dob strings for existing Client and Admin objects to have this format.  Anything "new" created uses this format.

### CODE REVIEW

File: Admin-app/src/components/DropdownDoctor/index.js [hunk 1]
```diff
@@ -148,10 +148,10 @@ function DropdownDoctor(props) {
             Object.values(doctorsList).map((doctor) => (
               <MenuItem
                 className="PATIENT__table__data"
-                value={doctor.id}
+                value={doctor.email}
                 disabled={isDoctorAtFullCapacity(doctor)}
               >
-                {doctor.name}
+                {`${doctor.firstName} ${doctor.lastName}`}
```
Comment (tusharraval102) [inline Admin-app/src/components/DropdownDoctor/index.js:154]: Is this what we are displaying for the dropdown menu?

**Review by tusharraval102 — COMMENTED** [review]
I am done reviewing the code, I will now run the code and check it out as well.

**Comment by tusharraval102** [comment]
After running the code, I think the everything looks good. I do have a few things to suggest: 

• The doctor dropdown should be searchable and scrollable.
• Maybe we should not show the symptoms list when it is "undefined" and we can show a message instead (it might be cleaner)

**Comment by smarkandu** [comment]
> After running the code, I think the everything looks good. I do have a few things to suggest:
> 
> • The doctor dropdown should be searchable and scrollable. • Maybe we should not show the symptoms list when it is "undefined" and we can show a message instead (it might be cleaner)

This is a refactor to switch the tables, and not to improve features.  We can do that with another PR.

**Review by MarwaKhalid — APPROVED** [review]
Fixed the dashboard patient list, otherwise, I think it looks good, everything seems to be pulled from the db. There are 2 high vulnerabilities but I can't figure out where they are coming from?

File: Admin-app/src/components/DropdownDoctor/index.js (hunk 1 above)
Comment (smarkandu) [reply Admin-app/src/components/DropdownDoctor/index.js:154]: Yeah, thats what we had before too.

**Review by QuangTran304 — APPROVED** [review]
Looks good to me. I tested some functions in `backend/firebaseAdminUtilities.js`. Seems to work well, so I trust that other utility functions works in the same fashion.

Also, since the Date format is now slightly changed in SignUp ==> that means once we merge, we'll have to update the Date format in Editable Modal for PatientProfile as well, right? @smarkandu 

It's nice that all the database querying functions are grouped in a common file but it still doesn't help avoiding calling the backend multiple times because we're still sending requests to the backend from inside components. I just recently made the 'store' for Admin-app. It basically send requests AND save the data fields from the response ==> so that we can reuse those fields without additional requests to our server.

**Pull request merged**